src/plugins/Dashboard/index.js

```diff
@@ -277,6 +277,8 @@ module.exports = class Dashboard extends Plugin {
         name: file.name,
         type: file.type,
         data: blob
+      }).catch(() => {
```

Comment:
> UI plugins swallow rejection errors so they don't end up in the console with no way to fix

But it will still catch upstream in `addFile` and restrictions to show the informer?.. 🤔